Userland/Libraries/LibConfig/Client.cpp

```diff
@@ -57,22 +57,22 @@ bool Client::read_bool(StringView domain, StringView group, StringView key, bool
 
 void Client::write_string(StringView domain, StringView group, StringView key, StringView value)
 {
-    async_write_string_value(domain, group, key, value);
+    write_string_value(domain, group, key, value);
 }
 
 void Client::write_i32(StringView domain, StringView group, StringView key, i32 value)
 {
-    async_write_i32_value(domain, group, key, value);
+    write_i32_value(domain, group, key, value);
 }
 
 void Client::write_bool(StringView domain, StringView group, StringView key, bool value)
 {
-    async_write_bool_value(domain, group, key, value);
+    write_bool_value(domain, group, key, value);
 }
 
 void Client::remove_key(StringView domain, StringView group, StringView key)
 {
-    async_remove_key(domain, group, key);
+    remove_key(domain, group, key);
```

Comment:
This calls itself, which is not what we want here.

Reply:
That was a bad mistake.
Instead of calling the right method. I figured that remove_key() ipc call is only used by QuickLaunchWidget for removing apps from Taskbar, and since Taskbar is a forever-running app it will not be affected by the aforementioned issue. Thus, we might better leave it as an async call.